toolkit/smart-contracts/offchain/src/csl.rs

```diff
@@ -368,7 +368,7 @@ impl UtxoIdExt for UtxoId {
 
 pub(crate) struct TransactionContext {
 	/// This key is added as required signer and used to sign the transaction.
-	pub(crate) payment_key: PrivateKey,
+	pub(crate) payment_key: CardanoPaymentSigningKey,
```

Comment:
This is changed because `CardanoPaymentSigningKey` has `Clone` but `PrivateKey` does not.